### PR TITLE
SCR1 - 02.07.2020 - V1

### PR DESCRIPTION
**Pull request by ABAPGIT-ACC**
SCR1 - 02.07.2020 - V1
SCR1 changes has been completed - Please review the changes

### CODE REVIEW

File: src/zsales_cl_abapgit.clas.abap [hunk 1]
```diff
@@ -12,6 +12,7 @@ CLASS zsales_cl_abapgit DEFINITION
         matnr TYPE vbap-matnr,
         charg TYPE vbap-charg,
         werks TYPE vbap-werks,
+        zmeng TYPE vbap-zmeng,        "SCR1 - 02.07.2020 - V1
```
Comment (Sakthy-g) [inline src/zsales_cl_abapgit.clas.abap:15]: New Field ZMENG added - as per FD - Ok
Please add unit of measure field MEINS from VBAP - Review Comments

File: src/zsales_cl_abapgit.clas.abap [hunk 2]
```diff
@@ -67,7 +68,8 @@ CLASS ZSALES_CL_ABAPGIT IMPLEMENTATION.
              posnr,
              matnr,
              charg,
-             werks
+             werks,
+             zmeng                "SCR1 - 02.07.2020 - V1
```
Comment (Sakthy-g) [inline src/zsales_cl_abapgit.clas.abap]: New field ZMENG added - as per FD - Ok
Please add Unit of Measure MEINS from VBAP - Review comments

**Review by Sakthy-g — CHANGES_REQUESTED** [review]
Review comments added - New changes required

File: src/zsales_cl_abapgit.clas.abap (hunk 1 above)
Comment (ABAPGIT-ACC) [reply src/zsales_cl_abapgit.clas.abap:15]: Unit of Measure added - Please Review

File: src/zsales_cl_abapgit.clas.abap (hunk 2 above)
Comment (ABAPGIT-ACC) [reply src/zsales_cl_abapgit.clas.abap]: Unit of Measure added - Please Review

File: src/zsales_cl_abapgit.clas.abap [hunk 3]
```diff
@@ -69,7 +70,8 @@ CLASS ZSALES_CL_ABAPGIT IMPLEMENTATION.
              matnr,
              charg,
              werks,
-             zmeng                "SCR1 - 02.07.2020 - V1
+             zmeng,               "SCR1 - 02.07.2020 - V1
+             meins                "SCR1 - 02.07.2020 - V1
```
Comment (Sakthy-g) [inline src/zsales_cl_abapgit.clas.abap:74]: As per FD - Ok

File: src/zsales_cl_abapgit.clas.abap [hunk 4]
```diff
@@ -13,6 +13,7 @@ CLASS zsales_cl_abapgit DEFINITION
         charg TYPE vbap-charg,
         werks TYPE vbap-werks,
         zmeng TYPE vbap-zmeng,        "SCR1 - 02.07.2020 - V1
+        meins TYPE vbap-meins,        "SCR1 - 02.07.2020 - V1
```
Comment (Sakthy-g) [inline src/zsales_cl_abapgit.clas.abap:16]: As per FD - Ok

**Review by Sakthy-g — APPROVED** [review]
Changes Ok - Approved to merge to master

**Pull request merged**